grid-client/integration_tests/two_vms_same_network_test.go

```diff
@@ -36,38 +36,45 @@ func TestTwoVMsSameNetworkWithPublicIPV6(t *testing.T) {
 
 	nodeID := uint32(nodes[0].NodeID)
 
-	network := generateBasicNetwork([]uint32{nodeID})
+	network, err := generateBasicNetwork([]uint32{nodeID})
+	require.NoError(t, err)
+
+	myCeliumSeed1, err := workloads.RandomMyceliumIPSeed()
+	require.NoError(t, err)
 
 	vm1 := workloads.VM{
```

Comment:
use `generateGasicVM` here

Reply:
same for the other vm

Reply:
you can always update the returned vm if the function is not customized enough for your need